Source/Core/Core/IOS/IOSC.h

```diff
@@ -17,6 +17,84 @@ class PointerWrap;
 
 namespace IOS
 {
+enum class SignatureType : u32
+{
+  RSA4096 = 0x00010000,
+  RSA2048 = 0x00010001,
+  // XXX: Add support for ECC (0x00010002).
+};
+
+enum class PublicKeyType : u32
+{
+  RSA4096 = 0,
+  RSA2048 = 1,
+};
+
+#pragma pack(push, 4)
+struct SignatureRSA4096
+{
+  SignatureType type;
+  u8 sig[0x200];
+  u8 fill[60];
+  char issuer[0x40];
+};
+static_assert(sizeof(SignatureRSA4096) == 0x280, "Wrong size for SignatureRSA4096");
+
+struct SignatureRSA2048
+{
+  SignatureType type;
+  u8 sig[0x100];
+  u8 fill[60];
+  char issuer[0x40];
+};
+static_assert(sizeof(SignatureRSA2048) == 0x180, "Wrong size for SignatureRSA2048");
+
+struct SignatureECC
+{
+  SignatureType type;
+  u8 sig[60];
+  u8 fill[64];
+  char issuer[0x40];
+};
+static_assert(sizeof(SignatureECC) == 0xc0, "Wrong size for SignatureECC");
+
+// Source: https://wiibrew.org/wiki/Certificate_chain
+struct CertHeader
+{
+  PublicKeyType public_key_type;
+  char name[64];
+  u32 id;
+};
+
+struct CertRSA4096
+{
+  SignatureRSA4096 signature;
+  CertHeader header;
+  // 0x100, not 0x200 bytes.
+  u8 public_key[0x100];
```

Comment:
That comments looks a little useless without context, is there anything you can add to it (like, why is it not `0x200`, and where does that number come from?)